src/rosetta-helpers.ts

```diff
@@ -169,3 +173,24 @@ function makePoisonMicroblockOperation(tx: DbMempoolTx | DbTx, index: number): R
 
   return sender;
 }
+
+export function publicKeyToBitcoinAddress(publicKey: string, network: string): string {
+  const publicKeyBuffer = Buffer.from(publicKey, 'hex');
+
+  let btcNetwork: btc.Network;
+  if (network == RosettaNetworks.mainnet) {
+    btcNetwork = btc.networks.bitcoin;
+  } else {
+    btcNetwork = btc.networks.regtest;
+  }
+
+  const address = btc.payments.p2pkh({
+    pubkey: publicKeyBuffer,
+    network: btcNetwork,
+  });
+  return address.address ? address.address : '';
```

Comment:
This is a failure condition correct? If so, it should throw rather than fail silently.

Reply:
This function is being called in a try-catch block that will handle if `p2pkh` fails. That being said this function should be autonomous. I will fix and update the PR.